docs-kits/kits/Data Chain Kit/Software Development View/page_software-development-view.md

```diff
@@ -16,6 +16,6 @@ The IRS Rest API provides an easy to use Interface for applications to access cr
 
 The IRS REST API is standardized through the Catena-X e.V. and can be found here:
 
-- [> CX-0005 Item Relationship Service API](https://catena-x.net/de/standard-library)
+- [> CX-0005 Item Relationship Service API](https://eclipse-tractusx.github.io/item-relationship-service/docs/swagger-ui/index.html)
```

Comment:
This URL should point to the IRS Standard Document CX-0005.
Please add the Link to 
`docs-kits/kits/Data Chain Kit/Software Development View/Job Api/irs-api.info.mdx`